src/test/java/io/github/mfvanek/pg/settings/PgParamImplTest.java

```diff
@@ -64,26 +61,21 @@ void equalsAndHashCode() {
         final PgParam theSame = PgParamImpl.of("statement_timeout", "2s");
         final PgParam second = PgParamImpl.of("lock_timeout", "2s");
 
-        assertNotEquals(first, null);
+        assertThat(first).isNotNull();
         //noinspection AssertBetweenInconvertibleTypes
-        assertNotEquals(first, BigDecimal.ZERO);
+        assertThat(BigDecimal.ZERO).isNotEqualTo(first);
 
         // self
-        assertEquals(first, first);
-        assertEquals(first.hashCode(), first.hashCode());
+        assertThat(first).isEqualTo(first);
+        assertThat(first.hashCode()).isEqualTo(first.hashCode());
 
         // the same
-        assertEquals(first, theSame);
-        assertEquals(first.hashCode(), theSame.hashCode());
+        assertThat(theSame).isEqualTo(first);
+        assertThat(theSame.hashCode()).isEqualTo(first.hashCode());
 
         // others
-        assertNotEquals(first, second);
-        assertNotEquals(first.hashCode(), second.hashCode());
-
-        // another implementation of PgParam
-        final PgParam pgParamMock = Mockito.mock(PgParam.class);
```

Comment:
Please restore deleted code

Reply:
this cannot be done, as assertion with mock in AssertJ is not supported